robottelo/ui/role.py

```diff
@@ -85,3 +85,29 @@ def add_permission(self, role_name, resource_type=None,
             self.click(tab_locators['roles.tab_location'])
             self.configure_entity(location, FILTER['filter_loc'])
         self.click(common_locators['submit'])
+
+    def get_resources(self, role_name):
```

Comment:
Oh, ok, got it to tests section, and can see your tests are using default roles, 'Viewer' included. In this case, please either check them manually and exclude 'Viewer' and others with >10 resources, or fetch total amount in a different way (there's total counter), or implement navigation through pages.